index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="header">
+      <nav class="container">
+        <div class="logo">
+          <a href="index.html">
+            <img src="./img/logo4.png" alt="" />
+          </a>
+        </div>
+        <div class="navigations">
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <a class="contact" href="#">Contact Us</a>
+          </ul>
+        </div>
+        <span class="material-symbols-outlined hamburger-menu">menu</span>
+      </nav>
+    </section>
+
+    <section class="welcome">
+      <div class="container welcome-details">
+        <div class="h1-div">
+          <h1>
+            2<sup>nd</sup> Global Submit on Applied Science, Engineering and
+            Technology<br />(GSASET)2023
+          </h1>
+        </div>
+        <div class="welcome-description">
+          <p>
+            The three days of the conference will contribute to build long-term
+            profitable and sustainable communication among the participants in a
+            wide variety of scientific areas with the common interest in Applied
+            Science, Engineering and Technology.
+          </p>
+        </div>
+        <h2 class="date">2023.03.23(TUE) ~ 25(WED)</h2>
+        <p class="welcomeP">@SHAGRI-LA HOTEL, BANGKOK, THAILAND</p>
+      </div>
+    </section>
+
+    <section class="main-events">
```

Comment:
- [ ] Kindly add an article to this section you are supposed to have five articles and you have 4 articles below is a comparison with the design given:
## desktop

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/196749327-7aecf894-d87e-4372-8a1f-ea8614c470d4.png) |  ![image](https://user-images.githubusercontent.com/61408860/196749395-b9215e36-8873-44df-8dce-02c8cb5afd31.png)

## mobile

- [ ] Kindly ensure the line text at the bottom matches the design given below is a comparison:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/196756240-4cf9f5dc-eabe-4a96-b922-30bd5eb76ce3.png) | ![image](https://user-images.githubusercontent.com/61408860/196756340-7a3759a0-60c7-41e8-9374-e7188ae42ca3.png)